packages/custom-token-bridging/scripts/exec.js

```diff
@@ -0,0 +1,181 @@
+const { providers, Wallet } = require('ethers')
+const { ethers } = require('hardhat')
+const { Bridge, networks } = require('arb-ts')
+const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
+require('dotenv').config()
+requireEnvVariables(['DEVNET_PRIVKEY', 'L2RPC', 'L1RPC'])
+
+/**
+ * Set up: instantiate L1 / L2 wallets connected to providers
+ */
+
+const walletPrivateKey = process.env.DEVNET_PRIVKEY
+
+const l1Provider = new providers.JsonRpcProvider(process.env.L1RPC)
+const l2Provider = new providers.JsonRpcProvider(process.env.L2RPC)
+ 
+const l1Wallet = new Wallet(walletPrivateKey, l1Provider)
+const l2Wallet = new Wallet(walletPrivateKey, l2Provider)
+
+/**
+* Set the initial supply of L1 token that we want to bridge
+*/
+const premine = ethers.utils.parseEther("10")
```

Comment:
In case anyway wants to actually run this against rinkeby, can we require a bit less?